Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -371,7 +368,9 @@ - (void)searchDisplayController:(SearchDisplayController *)controller updateSear
 
 - (void)searchDisplayControllerWillBeginSearch:(SearchDisplayController *)controller
 {
-    // NO-OP
+    // Note: We avoid switching to SearchMode in `shouldBegin` because it might cause layout issues!
```

Comment:
TIL → looks like it was just happening in iOS 13, ehh?

Reply:
Aye! I figured calling `reloadData` immediately before a layout cycle (triggered in **SearchDisplayController**) was a bit glitchy.